examples/experimental/PATE Analysis.ipynb

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
This notebook is empty, did you want to add an example of the code you produced, or didn't want to add this nb?

Reply:
deleted the notebook